src/private/CBCRuntime.m

```diff
@@ -28,20 +28,38 @@
 
 #pragma mark Primary demo check
 
-BOOL CBCCatalogIsPrimaryDemoFromClass(Class aClass) {
-  BOOL isPrimaryDemo = NO;
-
-  if ([aClass respondsToSelector:@selector(catalogIsPrimaryDemo)]) {
+void *CBCCatalogInvokeFromClassAndSelector(Class aClass, SEL selector) {
```

Comment:
Expose the `void *` in the arg here so that the calling function can provide the proper pointer type.